StaticDemo.cls

```diff
@@ -0,0 +1,13 @@
+public class StaticDemo {
+	StaticDemo s;
+	public static void staticMethod()
+	{
+		system.debug('In static method');
+	}
+	public void nonStaticMethod()
```

Comment:
@Eternus-noopur The question is `Write a static method in a class and call it from another (nonstatic)
class` which means you need to call this static method from another class not from same class.
- [ ] Update code as per question.